src/main/java/com/kodekonveyor/market/project/UpdateProjectModelController.java

```diff
@@ -29,34 +38,77 @@
   MilestoneEntityRepository milestoneEntityRepository;
   @Autowired
   AuthenticatedUserService authenticatedUserService;
+  @Autowired
+  MarketUserEntityRepository marketUserEntityRepository;
+  @Autowired
+  UpdateTasksService updateTasksService;
+  @Autowired
+  TaskEntityRepository taskEntityRepository;
 
   @Autowired
   Logger logger;
 
   @PutMapping(UrlMapConstants.UPDATE_PROJECT_MODEL_PATH)
   public ProjectDTO
-      call(final ProjectModelDTO projectModelDTO, final String projectName) {
+      call(final ProjectModelDTO projectModelDTO, final String projectName) throws JSONException {
```

Comment:
Please rethrow the JSONException as a validationException with a message understandable for the end user.

Reply:
due to some recent changes, seems like throwing the exceptions isn't needed, its being removed now.